common/commentable-model.js

```diff
@@ -43,7 +42,7 @@ export const CommentableModel = Base => class extends Base { // eslint-disable-l
      */
     commentCount() {
         // Necessary  for backwards compatibility with old comments
-        return _.isArray(this._commentCount) ? this._commentCount.length : this._commentCount || 0;
+        return Array.isArray(this._commentCount) ? this._commentCount.length : this._commentCount || 0;
```

Comment:
This can just return `this._commentCount`

Reply:
Should this method then be depracated?

Reply:
I don't see any reason why it can't. I think though that it might be a good Idea to at least change the underlying database fields in this release.. I think once the redis-oplog integration is complete we may see a bit more attention and more people using the packages.. Better to break things now when it affects the least amount of people.